javascript/builtins/intl/NumberFormat.json

```
@@ -195,16 +195,26 @@
                   "opera_android": {
                     "version_added": "55"
                   },
-                  "safari": {
-                    "version_added": "10",
-                    "partial_implementation": true,
-                    "notes": "Doesn't support <code>currencyDisplay: 'narrowSymbol'</code>."
-                  },
-                  "safari_ios": {
-                    "version_added": "10",
-                    "partial_implementation": true,
-                    "notes": "Doesn't support <code>currencyDisplay: 'narrowSymbol'</code>."
-                  },
+                  "safari": [
+                    {
+                      "version_added": "14.1"
+                    },
+                    {
+                      "version_added": "10",
```

If the 14.1 implementation supports `currencyDisplay: 'narrowSymbol'`, then…

```suggestion
                      "version_added": "10",
                      "version_removed": "14.1",
```

(otherwise, we need some more information about what the 14.1 support statement means)

---

Addressed in d766019.